Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -34,30 +34,35 @@ export class dotNetExe {
         tl.setResourcePath(path.join(__dirname, "task.json"));
         this.setConsoleCodePage();
 
-        switch (this.command) {
-            case "build":
-            case "publish":
-            case "run":
-                await this.executeBasicCommand();
-                break;
-            case "custom":
-                this.command = tl.getInput("custom", true);
-                await this.executeBasicCommand();
-                break;
-            case "test":
-                await this.executeTestCommand();
-                break;
-            case "restore":
-                await restoreCommand.run();
-                break;
-            case "pack":
-                await packCommand.run();
-                break;
-            case "push":
-                await pushCommand.run();
-                break;
-            default:
-                tl.setResult(tl.TaskResult.Failed, tl.loc("Error_CommandNotRecognized", this.command));
+        try {
+            switch (this.command) {
+                case "build":
+                case "publish":
```

Comment:
in case of publish command failure we should print the info message as warning before printing error message